src/OrbitGl/App.cpp

```diff
@@ -1189,9 +1188,8 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> OrbitApp::LoadCaptureFro
     if (capture_file_or_error.has_value()) {
       load_result = LoadCaptureFromNewFormat(this, capture_file_or_error.value().get(),
                                              &capture_loading_cancellation_requested_);
-    } else {  // Fall back to old capture format.
-      load_result = orbit_client_model::capture_deserializer::Load(
-          file_path, this, module_manager_.get(), &capture_loading_cancellation_requested_);
+    } else {  // The old format is not supported anymore.
+      load_result = ErrorMessage("Old captures are not supported anymore.");
```

Comment:
We should probably use capture_file_or_error.error() here because it provides more context.

Reply:
Makes sense. Done!